scripts/xml_validator.py

```diff
@@ -0,0 +1,15 @@
+import glob
+from lxml import etree
+
+parser = etree.XMLParser()
+XML_files = glob.glob("proxies/**/*.xml", recursive=True)
+
+for file in XML_files:
+    try:
+        etree.parse(file, parser)
```

Comment:
I guess this parsing will mostly lint completing static analysis rather than format? Did you mention that there wasn't really any XML formatters available?